src/website/app/App.js

```diff
@@ -26,12 +26,12 @@ import Footer from './Footer';
 import _Nav from './Nav';
 import Router from './Router';
 
-type Props = {|
```

Comment:
Curious why you removed the strict bars from all of the website components.

Reply:
They cause flow errors in some places when we are passing prop objects to components when those objects themselves are not strictly typed to match the props of the receiving component.

Also, the root prop spreading pattern that we use in the mineral components is useful and I don't see any reason why not to follow that pattern in the website.  Do you?

Reply:
No, I agree with your choice. :+1: I was just curious why. It's nice to have the same pattern everywhere, too.